panel/tests/chat/test_interface.py

```diff
@@ -215,6 +217,34 @@ def post_callback(instance, event):
         assert chat_interface.objects[0].object == "1"
         assert chat_interface.objects[1].object == "2"
 
+    def test_button_properties_default_callback_and_post_callback(self, chat_interface):
+        def post_callback(instance, event):
+            instance.send("This should show", respond=False)
+
+        chat_interface.button_properties = {
+            "clear": {"post_callback": post_callback},
+        }
+        clear_button = chat_interface._input_layout[-1]
+        chat_interface.send("This shouldn't show up!", respond=False)
+        clear_button.param.trigger("clicks")
+        time.sleep(0.5)
```

Comment:
Should have reviewed this more closely, `time.sleep` should not be used in the test suite (except for very rare cases). Use the `wait_until` utility.